crates/fluvio-spu/src/replication/test.rs

```diff
@@ -56,6 +56,8 @@ pub(crate) struct TestConfig {
     base_dir: PathBuf,
     #[builder(setter(into), default = "9000")]
     base_port: u16,
+    #[builder(setter(into), default = "2_097_152")]
```

Comment:
default is too big.  have specific test case to change this

Reply:
this is the default, I don't even use it, I can removed it